libs/top_level_metrics.py

```diff
@@ -23,8 +23,9 @@
 RT_TRUNCATION_DAYS = 7
 
 
-# CMS and HHS testing data can both lag by more than 7 days. Let's use it unless it's 2 weeks old.
-MAX_METRIC_LOOKBACK_DAYS = 13
+# CMS and HHS testing data can both lag by more than 7 days. Let's use it unless it's >2 weeks old.
+# TODO(michael): Consider having different lookback values per metric, but this is fine for now.
```

Comment:
And somewhat related, calculate_or_copy_test_positivity does its own thing with has_data_in_past_10_days